src/game_picture.cpp

```diff
@@ -204,30 +204,30 @@ void Game_Picture::Update() {
 	if (data.time_left == 0) {
 		SyncCurrentToFinish();
 	} else {
-		auto interpolate = [=](double& current, double finish) {
+		auto interpolate = [=](double current, double finish) {
 			double d = data.time_left;
-			current = (current * (d - 1) + finish) / d;
+			return (current * (d - 1) + finish) / d;
 		};
 
-		interpolate(data.current_x, data.finish_x);
-		interpolate(data.current_y, data.finish_y);
-		interpolate(data.current_red, data.finish_red);
-		interpolate(data.current_green, data.finish_green);
-		interpolate(data.current_blue, data.finish_blue);
-		interpolate(data.current_sat, data.finish_sat);
-		interpolate(data.current_magnify, data.finish_magnify);
-		interpolate(data.current_top_trans, data.finish_top_trans);
-		interpolate(data.current_bot_trans, data.finish_bot_trans);
+		data.current_x = interpolate(data.current_x, data.finish_x);
+		data.current_y = interpolate(data.current_y, data.finish_y);
+		data.current_red = interpolate(data.current_red, data.finish_red);
+		data.current_green = interpolate(data.current_green, data.finish_green);
+		data.current_blue = interpolate(data.current_blue, data.finish_blue);
+		data.current_sat = interpolate(data.current_sat, data.finish_sat);
+		data.current_magnify = interpolate(data.current_magnify, data.finish_magnify);
+		data.current_top_trans = interpolate(data.current_top_trans, data.finish_top_trans);
+		data.current_bot_trans = interpolate(data.current_bot_trans, data.finish_bot_trans);
```

Comment:
And the change to _interpolate_ was made because data.[fields] can't be taken by reference anymore with vNext.